api/src/models/project-survey-attachments.ts

```diff
@@ -67,15 +76,18 @@ export class GetReportAttachmentMetadata {
   description: string;
   year_published: number;
   revision_count: number;
+  security_review_timestamp: string;
```

Comment:
Let's make a `security` property which stores `security_review_timestamp` and `security_reasons`.